src/outline/manager.js

```diff
@@ -416,23 +416,23 @@ export default function (context) {
       .concat(pods)
 
     async function getPods () {
-      async function addPodRoot (pod) { // namedNode
-        await checkForContainerRepresentation(pod)
+      async function addPodStorage (pod) { // namedNode
+        await loadContainerRepresentation(pod)
         if (kb.holds(pod, ns.rdf('type'), ns.space('Storage'), pod.doc())) {
           pods.push(pod)
           return true
         }
         return false
       }
-      async function addPodRootFromUrl (url) {
+      async function addPodStorageFromUrl (url) {
         const podStorage = new URL(url)
-        // check that origin is not included in pods array
-        if (!pods.some(pod => pod.uri.includes(podStorage.origin))) {
-          // check in order, location.origin/<podName> then location.origin containing space:Storage
-          if (podStorage.pathname === '/' || !(await addPodRoot(kb.sym(`${podStorage.origin}/${podStorage.pathname.split('/')[1]}/`)))) {
-            await addPodRoot(kb.sym(`${podStorage.origin}/`))
-          }
+        // check for predicate pim:Storage in containers up the path tree
+        let pathStorage = podStorage.pathname
+        while (pathStorage.length) {
+          pathStorage = pathStorage.substring(0, pathStorage.lastIndexOf('/'))
+          if (await addPodStorage(kb.sym(`${podStorage.origin}${pathStorage}/`))) return
```

Comment:
question: I'm curious to know the difference between the storage paths where there is an index.html.  Also, why the change from root to storage.  Will be moving away from root and have a separate storage "directory"?

Reply:
The previous code was not loading the folder representation when the authentification was done from a window.location.href that was not the podRoot. This situation occurs always for pod with index.html at root.

Reply:
I have tested it using your profile link https://bourgeoa.solidcommunity.net:8443/profile/card#me, then logging in as me (my test server account) clicking storage.  It works for me. I can click between your storage and mine as well and it works very well!  Nice job!